src/components/teampage/team-members/teams-styles.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components"
+import hex from "../../../images/blue_hex.svg"
+import { HName, HUpper } from "../../../constants/typography"
+import { navyBlue, royalBlue } from "../../../constants/colors"
+import { size } from "../../../constants/devices"
+
+const TeamContainer = styled.div`
+  width: 86%;
+  margin: 0 auto;
+  padding-bottom: 16%;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    padding-bottom: 30vw;
```

Comment:
We can probably reduce this based on the design. This is how it looks on my phone so maybe 20vw ?
<img src="https://user-images.githubusercontent.com/60420625/116790371-c22a7e80-aa81-11eb-95dd-32539ba9ad5f.jpeg" alt="team page" width="300" />